database/migrations/2023_10_17_112549_create_form_builder_sections.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    public function up(): void
+    {
+        Schema::create('form_builder_sections', function (Blueprint $table) {
+            $table->id();
+            $table->uuid();
+            $table->foreignId('form_id')->constrained('form_builder_forms');
+            $table->string('name')->nullable();
+            $table->unsignedInteger('ordering')->default(0);
+            $table->unsignedInteger('columns')->default(1);
+            $table->text('description')->nullable();
```

Comment:
description after name, to keep it consistent

Reply:
done